src/Sulu/Bundle/MarkupBundle/Markup/HtmlTagExtractor.php

```diff
@@ -63,7 +63,7 @@ public function extract($html)
             $sortedTags[$name][$tag] = array_filter(array_merge($attributes, ['content' => $content]));
         }
 
-        return $sortedTags;
+        return [$this->namespace => $sortedTags];
```

Comment:
I'm a little bit worried about this... Adding another nesting level is something hard to grasp when working with any of the classes involved here. Wouldn't you think it is cleaner to return something like `TagMatch[]`? The `TagMatch` class could then have the namespace a variable in it.